set.mm

```diff
@@ -730632,6 +731072,1417 @@ last digit of the integer part (for ` b = ` 10 the first digit before
+      XIWUGUXHWUFQVWIVWPWUFUXHXHVWIWUFVWOUXHVWIWUFUXKVURSMZVUTVURSMZNMWUIWUJWFZ
+      PMVWOVWIUXKVUTVURVYMVYTWUAWUHYDVWIWUIWUJVWIUXKVURVYMWUAWUHYEVWIVUTVURVYTW
+      UAWUHYEYGVWIWUIVWKWUKVWNPVWIUXKVURVYMWUAWUHXMVWIWUKVUTVURWFZSMVUTVWLSMVWN
+      VWIVUTVURVYTWUAWUHUVLVWIWULVWLVUTSVWILVUQVYNVYPYHXLVWIVUQUXMVWLVYPVYSVWIV
+      UQLVYPVYNWRVUFVUHVWLRVOUXFVWHVVEVUQLVXCVVTVVELVUQVXTXNVRWBYFYIWNYJWOXJXGX
+      JXDYKYLYMYQUVMYNYOYMVUMVUIVUEVUMVUIVAZUYFUYOVUDWUMUYDUXQEUXIUYEVUMVUIUSUX
+      RUXIQZUYDUXQWKWUMWUNUYCUXPDUWQWUNUYBUXOUXHWUNUXTUXLUYAUXNPWUNUXSUXJUXKOUX
+      RUXILNWLTUXRUXIUXMOWMWNWOWPWQVUGUXQVUIYPYQUVNYRVUGUXQVUJVUEVCVUGVUJUXQVUE
+      VUGVUJUXQVUEVUGVUJVAZUXQVAZVUDUYFUYOWUPVUBUXMLLUXISMZNMZUXKOMZWUQUXHOMZPM
+      ZQZDUWQUIZFWUQVUCVUFVUJWUQVUCJZUXFUXQVUFVUJVAZWVDWUQKJZRWUQUOUPZWUQLWCUPZ
+      UGZWVEWVFWVGWVHWVELUXIWVEURZVUFVUJVIZWVEUXIWVERLUXIWVEUQWVJWVKVVNWVEVKVFV
+      UFVUJUSVLZXAVSZWVEUXIWVKWVLUVOWVEWVGWUQLUOUPZVAZWVHUXIUVPWVEWVOVAZWUQLWVE
+      WVFWVOWVMXSWVPURWVEWVGWVNUVQWHUVRUVSVVDVULVAWVDWVIWKWVEVVDVULVEUVTUWARLWU
+      QUWBUWFWGWBUYPWUQQZVUBWVCWKWUPWVQVUAWVBDUWQWVQUYTWVAUXMWVQUYRWUSUYSWUTPWV
+      QUYQWURUXKOUYPWUQLNWLTUYPWUQUXHOWMWNWOWPWQWUOUXQWVCWUOUXPWVBDUWQWUOVWHVAZ
+      UXPUXHWUQLNMZWUQSMZUXKOMZLWUQSMZUXMOMZPMZQZWVBWVRUXPWWEWVRUXOWWDUXHWVRUXL
+      WWAUXNWWCPWVRUXJWVTUXKOWVRWVTWUQWUQSMZWWBNMUXJWVRWUQLWUQWVRUXIWUOVVHVWHWU
+      OUXIUXFVUFVUJYPVJZXSZWUOUXIRVOZVWHVUGVUJWWIVUFVUJWWIVCUXFVUFVUJRUXIUOUPZW
+      WIVUFVUJWWJWVLYRVUFWWJWWIUXIUWCYRUWGWQYMZXSZWSZWVRWEZWWMWVRUXIWWHWWLUWDZY
+      DWVRWWFLWWBUXINWVRWUQWWMWWOUWEWVRUXIWWHWWLYSWNXOTWVRUXIWWBUXMOWUOUXIWWBQV
+      WHWUOWWBUXIWUOUXIWWGWWKYSYTXSTWNWOXPWVRWVBUXMWUSNMZWUTQZWWPWUQSMZUXHQZWWE
+      WVRWWQWVAUXMQWVBWVRUXMWUSWUTWVRUXMWUOUWQKUXGUWTUXFVYQVUFVUJVYRXTYAVJZWVRW
+      URUXKWVRLWUQWWNWWMWRZWVRUXKWUOUWQKUXGUWPUXFVYKVUFVUJVYLXTYAVJZYBZWVRWUQUX
+      HWWMWVRUXHWUOUWQKUXGUXDUXFWUBVUFVUJWUCXTYAVJZYBXCUXMWVAXHYCWVRWWSWUTWWPQW
+      WQWVRWWPWUQUXHWVRUXMWUSWWTWXCWRWWMWXDWWOXIWWPWUTXHYCWWSUXHWWRQWVRWWEWWRUX
+      HXHWVRWWRWWDUXHWVRWWRUXMWUQSMZWUSWUQSMZNMWXEWXFWFZPMZWWDWVRUXMWUSWUQWWTWX
+      CWWMWWOYDWVRWXEWXFWVRUXMWUQWWTWWMWWOYEWVRWUSWUQWXCWWMWWOYEYGWVRWXHWXEWWAP
+      MWWCWWAPMWWDWVRWXGWWAWXEPWVRWXGWUSWFZWUQSMWURWFZUXKOMZWUQSMZWWAWVRWUSWUQW
+      XCWWMWWOUWHWVRWXIWXKWUQSWVRWXKWXIWVRWURUXKWXAWXBUWIYTTWVRWXLWVSUXKOMZWUQS
+      MWWAWVRWXKWXMWUQSWVRWXJWVSUXKOWVRLWUQWWNWWMYHTTWVRWVSUXKWUQWVRWUQLWWMWWNW
+      RZWXBWWMWWOYFXKYIXLWVRWXEWWCWWAPWVRUXMWUQWWTWWMWWOXMTWVRWWCWWAWVRWWBUXMWV
+      RWUQWWMWWOWSWWTYBWVRWVTUXKWVRWVSWUQWXNWWMWWOYEWXBYBUWJYIYJWOXJXEYKYLYMYQU
+      WKYNYOYMUWLYRUWMUWN $.
+
+    $d N i k l m n p t v w x y z $.
+    $( The line definition in the Tarski structure for the Euclidean geometry
+       (see ~ elntg ) corresponds to the definition of lines passing through
+       two different points in a left module (see ~ rrxlines ).  (Contributed
+       by AV, 16-Feb-2023.) $)
```

Comment:
But elntg is not a definition ?
Ok, I searched and finally found it: https://us.metamath.org/mpeuni/df-trkg.html
@tirix : https://us.metamath.org/mpeuni/elntg.html says that it's a definition but it's not. It looks more like our *val theorems it seems. Also, the considered space is not made clear in the comment. So reword the comment to something like
> Expression of the line formed on two distinct points in the Tarski geometry formed on the real coordinate space of given dimension.

Reply:
Yes, some comments here:
The concept of "line" is not an elementary concept in Tarski's development; only betweenness and congruence are.

However, it is elementary in other developments of elementary geometry, like in Hilbert's axioms of geometry, or in incidence geometry. This is why a slot was added for Line, `LineG`. 

For Tarski's geometries, the way to define the `LineG` slot is part of its definition, and it is defined in terms of betweenness.

Reply:
Thanks for the explanation why it got a "Slot" (although I'm not sure it was worth it). Can you change the comment to something like I proposed above?
You can also add your comment above somewhere, but maybe it's already somewhere in set.mm: I only rapidly skimmed through it.

Reply:
It would be great if @avekens can do the change in this PR, however since `~elntg` is not changed by this PR, we might also consider it a different topic.

I'm putting this, the other point above about `~elntg` and other points left open in issue #3043 so they are not forgotten.

Reply:
Thanks @tirix for the new Issue !